.github/workflows/build.yml

```diff
@@ -2,24 +2,100 @@ name: Build
 
 on:
   workflow_call:
-    secrets:
-      BITBUCKET_READ_ONLY_SSH_KEY:
+    inputs:
+      AWS_ACCOUNT_NUMBER:
+        default: ''
+        description: 'The AWS Account number'
+        required: true
+        type: string
+      AWS_REGION:
+        default: 'us-west-2'
+        description: 'The AWS Region to deploy to'
+        required: false
+        type: string
+      BUCKET_IDENTIFIER:
+        default: ''
+        description: 'The S3 bucket identifier, typically the name of the repository'
+        required: true
+        type: string
+      BUSINESS_UNIT:
+        default: 'pmc'
+        description: 'The business unit'
+        required: false
+        type: string
+      ENVIRONMENT:
+        default: ''
+        description: 'The environment name'
         required: true
-      GITHUB_READ_ONLY_SSH_KEY:
+        type: string
+      IAM_ROLE_NAME:
+        default: ''
+        description: 'The AWS IAM Role to assume'
         required: true
+        type: string
+      S3_BUCKET_NAME:
+        default: ''
+        description: 'The S3 Bucket that the static assets are being deployed to'
+        required: true
+        type: string
 
-permissions:
```

Comment:
We should keep the `permission` block so that the GitHub token's scope is as narrow as is required for this step. Unless the step needs to commit to a branch or leave comments on the PR, `read` is sufficient.

Reply:
yup not sure why i removed this must have been by accident